locales/fr.json

```diff
@@ -308,6 +308,7 @@
     "global_settings_setting_example_int": "Exemple d’option de type entier",
     "global_settings_setting_example_string": "Exemple d’option de type chaîne",
     "global_settings_setting_example_enum": "Exemple d’option de type énumération",
+    "global_settings_setting_security_ciphers_compatibility": "Stratégie de compatibilité des ciphers",
```

Comment:
Sorry, that's not obvious but you should only edit en.json and not fr.json (or any other language) because supposedly weblate (our translation tool) might complain and stuff :s

Reply:
No problem. It's fixed.